docs/kaneai-release-notes-0-0-2.md

```diff
@@ -0,0 +1,100 @@
+
+## 4. Auto-Scroll for Manual Interaction 🎯
+No more searching for the latest entries after clicking! KaneAI now automatically scrolls to the latest interaction in manual testing sessions, saving time and making testing more efficient.
+
+## 5. View Issue Button on Error Only 🚨
+To enhance UI clarity, the "View Issue" button will only appear if there's an error within the viewport. This helps testers focus on relevant issues without unnecessary clutter.
+
+## 6. Screenshot Modal Fix 📸
+We’ve fixed an issue where screenshots would appear outside the modal on both KaneAI and TMS. Screenshots will now stay neatly within their designated area for better viewing.
+
+## 7. URL Update with Objective Changes 🌐
+Whenever you update an objective, the URL will now update along with any other pre-filled fields. This improvement ensures that your URLs stay in sync with your updates for better tracking and organization.
+
+## 8. Prevention of Empty API Calls 🛑
+We've added logic to prevent empty planner API calls on the main objective page. This optimization reduces unnecessary network activity and speeds up the experience.
+
+## 9. Label Overlap Fix in TMS 🏷️
+Fixed the issue of labels overlapping other elements on the TMS UI. Now, we show a concise label such as "+12" when there's no space. Clicking it will open an overlay with all label details, improving readability and UI neatness.
+
+## 10. Test Case Name Display Issue Fixed 🔍
+Resolved an issue where test case names weren't appearing in certain situations, such as when the screen was zoomed in at 200%. This fix ensures that test case names are visible even in extreme zoom conditions.
+
+## 11. Page Title Update 🔄
+We’ve officially rebranded all page titles from Auteur to KaneAI, making our identity consistent across all UI elements.
+
+## 12. Excluding KaneAI Test Cases from Test Plan & Build Creation ⚠️
+For better test management, we’ve restricted the creation of test plans and builds to exclude KaneAI Test Cases, ensuring a more streamlined process.
+
+## 13. Manual Interaction Enhancements 🖱️
+We've refined manual interactions to make Click actions faster and more precise, optimizing the overall experience for testers.
+
+## 14. Stable Template Support 📑
+Our templates have been stabilized for more reliable performance during test creation and execution.
+
+## 15. Appium Test Generation (Frontend Support) 📱
+While not live yet, we’ve added support for Appium test generation from the frontend, getting ready for future mobile testing capabilities.
+
+## 16. Meta Tag Updates for KaneAI 🏷️
+Meta tags have been updated for KaneAI, improving SEO and overall discoverability.
+
+## 17. Enhancements in Visual and Textual Query for Code Export 🔍
+We've enhanced the visual and textual query capabilities in code exports, making your scripts even more accurate when handling complex UI elements.
+
+## 18. UI Action Improvements 🎯
```

Comment:
This is part of code generation above. "UIActions" is the python file generated. Please remove.